### PR TITLE
[#135948443] Document lack of support for .NET Core buildpack

### PR DESCRIPTION
**Pull request by henrytk**
## What

Now that the default buildpacks offered by Cloud Foundry include .NET
Core and we have decided not to support it[1] we need to update our
documentation.

[1] https://www.pivotaltracker.com/n/projects/1275640/stories/135948443

## How to review

Read it. Check it is clear and correct.

## Who can review

Anyone but me.

### CODE REVIEW

**Comment by keymon** [comment]
I think we should also explain why we do not support it, it is a little bit surprising that there is one we do not support out of the blue.

**Comment by henrytk** [comment]
@keymon perhaps we should say if you want to use it to get in touch with us and we will probably be able to start supporting it?

**Comment by henrytk** [comment]
@keymon I have added another commit.

**Comment by keymon** [comment]
That makes more sense, because actually the reason why we do not support it is that we did not have any people asking for it. Although we did not have neither for other languages like java or php.

File: docs/getting_started/limitations.md [hunk 1]
```diff
@@ -2,21 +2,21 @@ While the Government PaaS is built using Cloud Foundry technology, we don't supp
...
 ## IP restrictions
 
-We currently only accept command line client connections from a list of approved IP addresses (a 'whitelist'). When you sign up to use the PaaS, you must tell us the IPs or IP ranges you and your developers will connect from, so we can add them to our whitelist. 
+We currently only accept command line client connections from a list of approved IP addresses (a 'whitelist'). When you sign up to use the PaaS, you must tell us the IPs or IP ranges you and your developers will connect from, so we can add them to our whitelist.
 
 You can view [the list of allowed IPs](https://github.com/alphagov/paas-cf/blob/master/terraform/prod.tfvars#L9). If you want us to allow additional IPs, please email us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
 
 Note that IP restrictions only apply to use of the command line client; there is no restriction on access to your deployed apps, which are available to everyone.
 
 ## Custom buildpacks are not supported
```
Comment (bleach) [inline docs/getting_started/limitations.md]: Maybe this should say "Custom buildpacks and the .NET buildpack are not supported"?

**Review by bleach — COMMENTED** [review]
Looks good. Let me know what you think about the heading.

File: docs/getting_started/limitations.md (hunk 1 above)
Comment (benh-gds) [reply docs/getting_started/limitations.md]: I've made the title "Custom buildpacks and .NET are not supported". I'll merge this.

**Pull request merged**